### PR TITLE
chore(index.d): correct parameter type for setTags

### PR DESCRIPTION
**Pull request by aidan-24com**
Hi,

I have picked up a minor typing discrepancy I believe. Please take a look and let me know if anything else is needed.

Thank you

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -58,4 +56,4 @@ declare module 'pushwoosh-react-native-plugin' {
 
   declare const Pushwoosh: Pushwoosh;
   export = Pushwoosh;
-}
```
Comment (aidan-24com) [inline index.d.ts:61]: this was automatically done by the github web editor. if it is a problem, let me know and I'll try and revert it

**Comment by wfhm** [comment]
@aidan-24com Thanks for your contribution! I believe replacing interface with a type is indeed a common and better solution, however, could you please clarify what exactly did you mean by the typing discrepancy? How exactly does it affect your app?

**Comment by aidan-24com** [comment]
Hello, @wfhm 

Our app is giving a linter error for the `setTags` function.

image


the typing discrepancy is the type of the `tags` parameter for the `setTags` function (line 26). If you look at the current release's code, the type is defined as `tags: Record<string, PushwooshTags>`. I might be wrong, but I think it is supposed to be just the `PushwooshTags` type, ie `tags: PushwooshTags`.

I took the opportunity to use the `Record<string, ...>` syntax, since it was originally used in the `setTags` definition.

I am more than willing to remove the changes to the `PushwooshTags` interface. The main point of this PR is to "correct" the typing change to the `setTags` function.

**Comment by wfhm** [comment]
@aidan-24com You are right, indeed the types do not match. Thanks again for pointing it out and creating this PR!

**Pull request merged**